airflow-core/src/airflow/ui/src/i18n/locales/de/README.md

```diff
@@ -62,6 +62,11 @@ Für die Deutsche Übersetzung wurden die folgenden Terme wie folgt übersetzt
 - `Asset Event` --> `Ereignis zu Datenset (Asset)`: Logische Konsequenz der
   Übersetzung von -->"Asset" ohne einen sperrigen Begriff wie
   "Datenssatz-Ereignis" zu erzeugen.
+- `Backfill` --> `Auffüllen`: Der technisch geprägte Term im Programm passt zu
+  dr direkten Übersetzung, auch im Deutschen werden Lücken wieder "aufgefüllt".
+- `Bundle` --> `Bündel`: Die direkte Übersetzung passt zu dem Ziel der
+  Englischen Begriffsdefinition.
+- `Catchup` --> `Nacholung`: Direkte Übersetzung.
```

Comment:
```suggestion
- `Catchup` --> `Nachholung`: Direkte Übersetzung.
```

Nachholung or "Nachholen" 🤔 I think the term is more commonly used as a verb in Airflow so I'd go with the latter, like with Backfill. In either case needs two hs :)